### PR TITLE
Add prettier code formatter

### PR DESCRIPTION
**Pull request by ziyuan-linn**
Add Prettier code formatter to keep the code consistent. Update README to include a guide on how to use it.

### CODE REVIEW

**Review by shiffman — DISMISSED** [review]
Looks great! I think the info on prettier can probably go in CONTRIBUTING.md?

**Comment by ziyuan-linn** [comment]
Sounds good! Moved the info for Prettier to CONTRIBUTING.md. Also added a duplicated setup section.

File: README.md [hunk 1]
```diff
@@ -1,37 +1,40 @@
 # ml5-neuralnetwork-test-build
+
 This is a test repo to see what it would take to build only the NeuralNetwork feature of the ml5 library.
```
Comment (shiffman) [inline README.md:3]:
```suggestion
Welcome to the next-generation repo for the ml5.js project! Due to the complexities inherent in updating all the dependencies in [the original repo](https://github.com/ml5js/ml5-library), we're exploring rebuilding from the ground up. This fresh repository allows us to migrate over classes and functions one at a time. This is very much a work in progress and is currently in its early, experimental stage. We appreciate your patience as we work towards refining this next evolution of the ml5.js project. Stay tuned!
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
-This will create the build in the dist folder.
 
+This will create the build in the dist folder.
 
 Open `examples/NeuralNetwork/index.html` in the browser to see the build running an example from the ml5 website.
 
-
 ## Process
+
+_This section explains how this repository was creates and is for reference purposes only. The steps outlines in this section do not need to be followed by contributors._
```
Comment (shiffman) [inline README.md:28]: @ziyuan-linn maybe this section also goes to CONTRIBUTING.md?

**Review by shiffman — DISMISSED** [review]
I wrote a paragraph for the main README and made a small additional comment. @gohai would you mind taking a look and then we can merge I think!

File: README.md (hunk 2 above)
Comment (ziyuan-linn) [reply README.md:28]: ok, I will move it to CONTRIBUTING.md

File: package.json [hunk 3]
```diff
@@ -13,9 +14,10 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/ml5js/ml5-neuralnetwork-test-build"
```
Comment (gohai) [inline package.json:17]: This wants to be https://github.com/ml5js/ml5-next-gen

Comment (gohai) [inline package.json:17]:
```suggestion
    "url": "https://github.com/ml5js/ml5-next-gen"
```

**Review by gohai — APPROVED** [review]
Looks very good to me!

**Pull request merged**